### DISCUSSION TITLE
A Call for Open Source Contributions on the XDC Network

### DISCUSSION THREAD

**Discussion by CoinClubQuincy** [XDC-Community/XDC-Community.github.io#18] [Ideas]
XDC Foundation has actively been building Open source Contracts for developers to expand upon in their applications, as they build upon the XDC network. We are actively encouraging developers to also contribute to the XDC Open Source Community to help build a better broader ecosystem of DApps, Libraries, and Tools to allow for a continuous growth of development on the network as well as have an easier, more productive time building applications for developers by utilizing the contributions of other developers in the xdc open source community. 

All contributions are  welcome as well as appreciated to help further the growth of the space!

**Comment by javvyDuke** [comment]
Hi @CoinClubQuincy i think I have the right idea that will increase the use case. I commented on your last post on twitter yesterday but I thought this may be the best way to reach out. I have been working on my white paper for a DApps for the last month and a half and I think I’m ready to bring it to the world.

**Reply by CoinClubQuincy** [reply]
Great!! I highly recommend joining our Developer discord to speak more on it and collaborate with other developers with similar goals:
https://discord.gg/VVr6pc6s

**Reply by cryptoRREN** [reply]
> 

were you able to join the Discord Server? The developer chat is a great place to kick off these discussions and Quincy can assist in this chat as well.
